modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/FlaskConnexionCodegen.java

```diff
@@ -317,6 +316,15 @@ public void preprocessSwagger(Swagger swagger) {
                                     controllerPackage + "." + toApiFilename(tag)
                             );
                         }
+                        for (Parameter param: operation.getParameters()) {
+                            // sanitize the param name but don't underscore it since it's used for request mapping
```

Comment:
That means the developers need to use the **updated** spec to generate the API clients, right? otherwise the auto-generated API client cannot talk to the Flask backend as the parameter name may have been renamed.